src/main/resources/assets/structurize/lang/en_us.json

```diff
@@ -1152,4 +1152,5 @@
   "block.structurize.horizontal_plain_cactus_birch_timber_frame": "Plain Horizontal Cactus Birch Timber Frame",
   "structurize.gui.buildtool.creative_only": "Structurize does not support using the build tool when in survival. Switch to creative or install MineColonies and use the MineColonies Builder.",
   "structurize.gui.replaceblock.ambiguous_properties": "Transformation from %s to %s is ambiguous because the following properties are not present in the source block: %s"
+  "structurize.gui.buildtool.unexpecteddatafixer": "Invalid datafixer detected, unpredicted side-effects possible, check log for more info!"
```

Comment:
```suggestion
  "structurize.gui.buildtool.unexpecteddatafixer": "Invalid datafixer detected! Unpredicted side-effects possible! Check the log for more info."
```

Reply:
`Unpredictable side effects are possible!`

Reply:
Needed to cut Unpredictable because of no space. It's now "Side-effects are possible